src/debugger/titaniumSourceMapTransformer.ts

```diff
@@ -22,25 +22,27 @@ export class TitaniumSourceMapTransformer extends BaseSourceMapTransformer {
 
 	public async configureOptions (args: TitaniumLaunchRequestArgs) {
 		this.appDirectory = args.projectDir;
-		this.platform = args.platform === 'ios' ? 'iphone' : args.platform;
+		this.platform = args.platform;
 		this.projectType = await determineProjectType(this.appDirectory);
 	}
 
 	public async scriptParsed (pathToGenerated: string, sourceMapUrl: string): Promise<string[]> {
 		if (this.projectType === 'alloy') {
-			const mapDir = path.join(this.appDirectory, 'build', 'map', 'Resources', this.platform);
+			const platform = this.platform === 'ios' ? 'iphone' : this.platform;
```

Comment:
Seeing as this isn't the really the proper platform but used in the sourcemaps, maybe something like `sourceMapPlatform`?